src/components/encoding-pane/property-editorSchema.ts

```diff
@@ -0,0 +1,79 @@
+import * as vlSchema from 'vega-lite/build/vega-lite-schema.json';
+
+export const AXIS_TITLE_SCHEMA = {
+  "type": "object",
+  "title": "Axis",
+  "properties": {
+    "axisTitle": {
```

Comment:
What's the naming convention for the properties of schema here? 

They seem inconsistent.  axisTitle, stackSelect, ... 

Define a consistent policy.  Add comment what's your policy at the top of the file.